rkemmy/food-first
# Feature user create order #160812191

**What does this PR do?**
Allows user to place an order for food

**Task Description**
* Check that user is logged in
* Check that input is valid
* Create order

**Manual Testing**
Open terminal then run:
```git clone https://github.com/rkemmy/food-first.git```

cd into project folder and create a virtual environment then run
```pip install -r requirements.txt```

Test the following endpoint
```POST   /api/v2/post```
set KEY = Content-Type , VALUE= application/json
JWT Token should be included
KEY = Authorization , VALUE = Bearer, JWT Token from Login endpoint
Press send

**Relevant User Stories**
[#160812191](https://www.pivotaltracker.com/story/show/160812191)
